### DISCUSSION TITLE
import textfile containing outline structure? (AsciiDoc, Markdown, or simple flat file)

### DISCUSSION THREAD

**Discussion by aisbergde** [freeplane/freeplane#1720] [Help]
freeplane has export to AsciiDoc and Markdown, is there a reverse import? Even a limited import?
In other words - Is it possible to import a map or a branch from a text file, containing an outline structure?
Or what could be the workflow? Maybe first converting into a format, which can be imported into freeplane?

**Comment by abc16361** [comment]
Copy the text and paste it to a node will do.
